features/video.yml

```
@@ -0,0 +1,77 @@
+name: <video>
+description: The `<video>` element embeds a video media player in a document.
```

Maybe...

```suggestion
description: The `<video>` element plays videos or movies, optionally with controls provided by the browser.
```

---

Continuing on my commentary above:

```suggestion
description: The `<video>` element represents a video file or stream, a moving picture. Optionally, the browser provides playback controls. 
```